design/specs/data-type-options.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Data Type Options
+description: 
+published: true
+date: 2022-09-01
+tags: 
+editor: markdown
+dateCreated: 2022-09-01
+---
+
+## Context
+
+Data types are used to define the type of data that a column can hold. Each data type has a set of options that can be configured to define how the data is interpreted and displayed. These set of options are split into two categories:
+
+- **Database**: These options are used to define how the data is stored in the database.
+- **Format**: Defines how the data is displayed to the user.
+
+## Problem
+
+The current implementation uses tabs to switch between the two categories of options. This is not ideal because data type options are used in Table Inspector, where tabs are already used to switch between different sections. We are trying to avoid having a tabbed interface within a tabbed interface.
+
+## Vertical Space Considerations
+
+The solution should consider the challenges in displaying the options in a small vertical space, such as the dropdown menu. Scroll bars should be avoided if possible.
+
+## Proposed Solution
+
+### Replace tabs with stacked sections
+
+Update Database and Format options to be stacked sections inside the Data Type section. The user should be able to access the options without the need of tabs. The styling should make the distinction between sections and sub-sections clear.
+
+### Use a dropdown menu for the data type selection instead of the scrollable list
+
+![image](/assets/design/specs/data-type-options/gL37Da5Tdp4iTZScWJPGC.png)
+
+### Align some of the label and field groups inline
+
+Inline alignment can be used for groups of fields in order to save vertical space.
+
+![image](/assets/design/specs/data-type-options/6VPzH8Cd7USbhf6jZpzQvc.png)
```

Comment:
I can see that the gain obtained in making the input labels inline and using a select menu instead of scrollable list would make up for the vertical space lost in getting rid of tabs.

I think we can proceed with these set of changes for now.